tests/plugins/menu/manual/rightclick.html

```diff
@@ -0,0 +1,12 @@
+<h2>Classic editor</h2>
+<div id="classic">
+	<p>I'm CKEditor 4 instance.</p>
+</div>
+
+<script>
+	if ( !CKEDITOR.env.ie ) {
```

Comment:
This test does not have any sense outside of IE, because it's not possible to activate button by right mouse button. In the IE it's possible only because `mouseup` is used there instead of `click`.

Reply:
TBH I'm quite sure that this kind of breakage can't be missed even without tests, as it would make any button unclickable. But added manual test for `panel` should also cover such cases.